internal/envelope/envelope.go

```diff
@@ -46,7 +46,7 @@ func GetEnvelopeMediaType(sigFormat string) (string, error) {
 	case COSE:
 		return cose.MediaTypeEnvelope, nil
 	}
-	return "", fmt.Errorf("signature format %q not supported", sigFormat)
+	return "", fmt.Errorf("signature format %q not supported\nPlease use the supported signature envelope format \"jws\" or \"cose\"", sigFormat)
```

Comment:
This looks good, but maybe we can make it a little bit more concise?
```suggestion
	return "", fmt.Errorf("signature format %q not supported\nSupported signature envelope formats are \"jws\" and \"cose\"", sigFormat)
```

Reply:
Both look good and user friendly to me. @JeyJeyGao 's description seems much more actionable and straightforward. to users.

Reply:
> Both look good and user friendly to me. @JeyJeyGao 's description seems much more actionable and straightforward. to users.

@FeynmanZhou Asked ChatGPT for suggestion:
```
Both options are grammatically correct and convey the same meaning, but the second option is more concise and easier to understand. Therefore, I would recommend using the second option: "Supported signature envelope formats are 'jws' and 'cose'."`
```
(the question was 'which one is better for an error message?')

Reply:
Updated.